src/mon/HealthMonitor.h

```diff
@@ -45,9 +45,10 @@ class HealthMonitor : public QuorumService
   virtual bool service_dispatch(MonOpRequestRef op);
 
   virtual void start_epoch() {
+    epoch_t epoch = get_epoch();
     for (map<int,HealthService*>::iterator it = services.begin();
          it != services.end(); ++it) {
-      it->second->start(get_epoch());
```

Comment:
On Sun, Jul 17, 2016 at 11:06:57PM -0700, song.baisen wrote:
> >      for (map<int,HealthService*>::iterator it = services.begin();
> >           it != services.end(); ++it) {
> > -      it->second->start(get_epoch());
> 
> @branch-predictor  Thank you for reply.I change the title.

Please change the commit message as well.